.github/workflows/release-e2e-workflow-template-windows.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       TERM: xterm
       # make Node run in ipv4 first so that cypress can detect 5601 port in CI environment
       NODE_OPTIONS: '--max-old-space-size=6144 --dns-result-order=ipv4first'
+      # 2.12 onwards security demo configuration require a custom admin password
+      OPENSEARCH_INTIAL_ADMIN_PASSWORD: 'myStrongPassword123!'
```

Comment:
For curious, is this the new default password of admin user? or security plugin will honer env variable `OPENSEARCH_INTIAL_ADMIN_PASSWORD`

Reply:
@derek-ho Same question here, could you clarify this?

Reply:
```suggestion
      OPENSEARCH_INITIAL_ADMIN_PASSWORD: 'myStrongPassword123!'
```

Reply:
Thanks for the catch! And yes, when running the install demo config we will honor whatever you put as that variable.